app/assets/javascripts/drop.js

```diff
@@ -159,6 +160,22 @@ SoapStone.DropView.prototype.showDrop = function (drop) {
   });
 };
 
+SoapStone.DropView.prototype.clearDropList = function (drop) {
+  $(".drop-list").remove();
+};
+
+SoapStone.DropView.prototype.showDropList = function (drops) {
+  var self = this;
+  $("[data-view='map']").append(this.dropListTemplate(drops));
+  $("[data-view='map']").on('click', 'a', function(event){
+    if (event.target.dataset.drop) {
+      event.preventDefault();
+      self.controller.showDrop(event.target.dataset.drop);
```

Comment:
event.target.dataset.drop 
this when over my head. if you say this works i believe you